index.html

```diff
@@ -23,165 +30,161 @@
         <a class="dropdown">
           <img src="./images/Dropdown.svg" alt="drop down icon" />
         </a>
+        <ul class="Section_link">
+          <li id="Portfolio_link"><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#"><img src="./images/D_Mail.svg" alt="Mail icon" class="mail_box"/></a></li>
+        </ul>
       </nav>
     </header>
     <div class="hov">Zahra's Portfolio</div>
     <!-- Headline Section -->
     <div class="headline">
-      <h1 class="mainText CreteRoundfont">Hey There. I’m Zahra</h1>
+      <h1 class="item-A mainText CreteRoundfont">Hey There. I’m Zahra</h1>
 
-      <p class="textcolor Robotofont">I’am a Software Developer</p>
-      <p class="textcolor Poppinsfont">
-        I can help you build a product , feature or website Look through some of
+      <p class="item-B textcolor Robotofont">I’am a Software Developer</p>
+      <p class="item-C textcolor Poppinsfont">
+        I can help you build a product, feature or website Look through some of
         my work and experience! If you like what you see and have a project you
         need coded, don’t hestiate to contact me.
       </p>
-      <ul class="contactIcons">
+      <ul class="item-D contactIcons">
         <li><img src="images/GitHub.svg" alt="" /></li>
         <li><img src="images/Linkedin.svg" alt="" /></li>
         <li><img src="images/Angelist.svg" alt="" /></li>
         <li><img src="images/Twitter.svg" alt="" /></li>
         <li><img src="images/Medium.svg" alt="" /></li>
       </ul>
-
-      <!-- <img src="./images/bdimage.svg" alt="backgroung image" class="bgimg"> -->
     </div>
 
     <!-- Works Section -->
     <section class="worksection">
       <h2 class="workheading CreteRoundfont">My Recent Works</h2>
       <hr />
```

Comment:
- [ ] Kindly make the line next to the header `My Recent Works` as the Figma design. You can use flex to make it.

Your design
![image](https://user-images.githubusercontent.com/35535130/166123296-c40e11d8-061e-4baf-9aac-9736bb280d0d.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123306-de27a8fe-9874-497a-9edc-bd42e9babb1f.png)